index.html

```diff
@@ -478,19 +478,17 @@ <h2>
           the <a>push subscriptions</a> it serves.
         </p>
         <p>
-          The <a>user agent</a> is responsible for choosing the <a>push service</a> used for
-          creating <a>push subscriptions</a>. There usually is a pre-existing relationship between
-          the <a>user agent</a> and the <a>push service</a> due to a variety of performance-related
-          concerns, including the complexity of running reliable <a>push services</a> and the impact
-          on battery lifetime if there were an unbounded set of <a>push services</a> to which a
-          device could connect.
+          The <a>user agent</a> connects to the <a>push service</a> used to create <a>push subscriptions</a>.
+          <a>User agents</a> MAY limit the choice of <a>push services</a> available. Reasons for doing so
+          Include performance-related concerns such as service availability (including whether services are
+          blocked by firewalls in specific countries, or networks at workplaces and the like), reliability,
+          the ability to control impact on battery lifetime, and agreements to steer metadata to, or away from,
+          specific <a>push services</a>.
         </p>
         <p class="note">
```

Comment:
Given that this is now in the security/private section, maybe we could drop this paragraph from here.

Reply:
Agreed. Done